src/components/dropdown/Dropdown.vue

```diff
@@ -268,6 +322,21 @@ export default {
         close() {
             this.filteredItems = this.items;
             this.toggle = false;
+        },
+        addItem() {
```

Comment:
In the `addItem` emit an event that indicates the action, so that it can be used later on for possibly adding values to the BE directly from it.